### PR TITLE
Implement design recommendations. New version 4.3 beta

### PR DESCRIPTION
**Pull request by aleixac**
New Plugin version 4.3.0 beta.
- Improved toolbar styles with arrow.
- Improved explanation tooltip and popup texts.
- Improve code with new coinform constants class.
- Added extra labels metadata.
- Tunning tweet queries retry times.
- Minimum adaptions to twitter dark mode page.
- Open after plugin installation welcome page.
- Implemented user options to disable the plugin nudging actions.
- Unblur/reblur popup button moved.
- Added coinform website explanation links.
- Added project and tool About panel.
- New messages and multilanguage translations.
- Added labels feedback numbers threshold code.
- Added assessment links in the misinformation Await popup.

### CODE REVIEW

File: src/js/browser-plugin.js [hunk 1]
```diff
@@ -486,7 +530,7 @@ const checkLabelMisinfo = (label) => {
     if (!labelCategory) {
       logger.logMessage(CoInformLogger.logTypes.warning, `Unexpected Label: ${label}`);
     }
-    else if (labelCategory.localeCompare("blur") === 0) {
+    else if (labelCategory.action.localeCompare("blur") === 0) {
       misInfo = true;
```
Comment (daffyDukk) [inline src/js/browser-plugin.js:534]: boolean variable names be prefixed with an 'is'

File: src/js/browser-plugin.js [hunk 2]
```diff
@@ -1217,20 +1224,47 @@ const createLabelModulesExplainabilityContent = (label, modules) => {
...
+  explainInfoPart2.append(explainInfoPart2Txt1);
+  let notVerifiableTxt = browserAPI.i18n.getMessage('not_verifiable');
+  let explainInfoPart2Txt2 = document.createTextNode(browserAPI.i18n.getMessage('not_verifiable_case_description', notVerifiableTxt));
+  explainInfoPart2.append(document.createTextNode(" "));
+  explainInfoPart2.append(explainInfoPart2Txt2);
+  explainInfoContent.append(document.createElement("BR"));
+  explainInfoContent.append(explainInfoPart2);
+
+  let explainInfoPart3 = document.createElement("SPAN");
+  let explainInfoPart3Txt = browserAPI.i18n.getMessage('more_generic_analysis_info__html', CoinformConstants.ANALYSIS_INFO_URL);
```
Comment (daffyDukk) [inline src/js/browser-plugin.js:1240]: Two '__' between 'info' and 'html'. Intended or misspelling?

File: src/js/browser-plugin.js [hunk 3]
```diff
@@ -1217,20 +1224,47 @@ const createLabelModulesExplainabilityContent = (label, modules) => {
...
 
       let moduleName = browserAPI.i18n.getMessage(key);
       let moduleBased = browserAPI.i18n.getMessage(`${key}_based_info`);
-      let moduleAnalysisInfoHtml = browserAPI.i18n.getMessage('module_analysis_long_info__html', [moduleName, moduleBased]);
+      let clickMoreInfo = browserAPI.i18n.getMessage(`click_more_info`);
+      let moduleUrl = "#";
+      if (CoinformConstants.MODULES_INFO_URLS[key.toUpperCase()]) {
+        moduleUrl = CoinformConstants.MODULES_INFO_URLS[key.toUpperCase()];
+      }
+      let moduleAnalysisInfoHtml = browserAPI.i18n.getMessage('module_analysis_long_info__html', [moduleName, moduleBased, moduleUrl, clickMoreInfo]);
```
Comment (daffyDukk) [inline src/js/browser-plugin.js:1267]: Two underscores here aswell

File: src/js/browser-plugin.js [hunk 4]
```diff
@@ -1525,18 +1580,27 @@ function updateLabelEvaluation(targetButton, tweetInfo, agreement) {
...
   let totalNum = 0;
   if (pluginCache[tweetInfo.id].feedback[`total_${agreement}`] != undefined) {
-    totalNum = pluginCache[tweetInfo.id].feedback[`total_${agreement}`];
+    totalNum = parseInt(pluginCache[tweetInfo.id].feedback[`total_${agreement}`]);
   }
-  if (operation == 'add') totalNum = totalNum + num;
-  else if (operation == 'remove') totalNum = totalNum - num;
-  else if (operation == 'update') totalNum = num;
-  if (totalNum >= 1) {
+  if (operation == 'add') totalNum = totalNum + parseInt(num);
```
Comment (daffyDukk) [inline src/js/browser-plugin.js:1592]: can do a += here instead of the long syntax

File: src/js/browser-plugin.js [hunk 5]
```diff
@@ -1525,18 +1580,27 @@ function updateLabelEvaluation(targetButton, tweetInfo, agreement) {
...
+  else if (operation == 'remove') totalNum = totalNum - parseInt(num);
```
Comment (daffyDukk) [inline src/js/browser-plugin.js:1593]: and a -= here

**Review by daffyDukk — DISMISSED** [review]
Some minor things and questions. Looks good to me

File: src/js/browser-plugin.js (hunk 1 above)
Comment (aleixac) [reply src/js/browser-plugin.js:534]: I agree, I'll update that

File: src/js/browser-plugin.js (hunk 2 above)
Comment (aleixac) [reply src/js/browser-plugin.js:1240]: Yep, is intended. My way to separate plain text messages than the ones that allows html tags. As we also use underscores in the messages keys, I decided to separate with the "html" notation using 2*underscores.

File: src/js/browser-plugin.js (hunk 3 above)
Comment (aleixac) [reply src/js/browser-plugin.js:1267]: Same as [here](https://github.com/co-inform/browser-plugin/pull/63#discussion_r674724555).

File: src/js/browser-plugin.js (hunk 4 above)
Comment (aleixac) [reply src/js/browser-plugin.js:1592]: I agree, I'll update that

File: src/js/browser-plugin.js (hunk 5 above)
Comment (aleixac) [reply src/js/browser-plugin.js:1593]: I agree, I'll update that

**Review by daffyDukk — APPROVED** [review]
Looks good to me

**Pull request merged**